src/InformaticsGateway/Services/Export/ExportServiceBase.cs

```diff
@@ -328,12 +347,12 @@ private void ReportingActionBlock(ExportRequestDataMessage exportRequestData)
                    _configuration.Export.Retries.RetryDelays,
                    (exception, timeSpan, retryCount, context) =>
                    {
-                       _logger.ErrorAcknowledgingMessageWithRetry(exception, timeSpan, retryCount);
+                       _logger.ErrorPublishingExportCompleteEventWithRetry(exception, timeSpan, retryCount);
```

Comment:
swapped these two around, so that the message  is acknowledged last, again to help with the fact that the next message will be received before the  Interlocked.Decrement is called